proto/ibc/applications/fee/v1/fee.proto

```diff
@@ -1,16 +1,18 @@
 syntax = "proto3";
 
 package ibc.applications.fee.v1;
+
 import "cosmos/base/v1beta1/coin.proto";
 import "gogoproto/gogo.proto";
 import "ibc/core/channel/v1/channel.proto";
+
 option go_package = "github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types";
```

Comment:
Small nit: move `option go_package` above imports to align with other IBC protos (should've included that in the doc earlier)

https://developers.google.com/protocol-buffers/docs/style#file_structure

Reply:
Done. Thanks